src/connectedk8s/azext_connectedk8s/commands.py

```diff
@@ -22,13 +22,14 @@ def load_command_table(self, _):
     )
     with self.command_group('connectedk8s', connectedk8s_sdk, client_factory=cf_connected_cluster) as g:
         g.custom_command('connect', 'create_connectedk8s', supports_no_wait=True)
-        g.custom_command('update', 'update_agents')
+        g.custom_command('update', 'update_agents', supports_no_wait=True)
```

Comment:
since it does not only mean to update_agents should we rename this method ?.

Reply:
renamed to update_agents_or_resource